earth_enterprise/src/third_party/apache2/apache-gee-httpd-ssl.conf.in.patch

```diff
@@ -0,0 +1,108 @@
+--- httpd-2.2.32.orig/docs/conf/extra/httpd-ssl.conf.in	2015-05-27 11:59:59.000000000 -0700
++++ httpd-2.2.32/docs/conf/extra/httpd-ssl.conf.in	2017-06-14 17:33:47.846665091 -0700
+@@ -56,7 +56,9 @@
+ #   ensure these follow appropriate best practices for this deployment.
+ #   httpd 2.2.30, 2.4.13 and later force-disable aNULL, eNULL and EXP ciphers,
+ #   while OpenSSL disabled these by default in 0.9.8zf/1.0.0r/1.0.1m/1.0.2a.
+-SSLCipherSuite HIGH:MEDIUM:!MD5:!RC4
++# Override SSL configuration to require more secure ciphers.
++#SSLCipherSuite HIGH:MEDIUM:!MD5:!RC4
++SSLCipherSuite ALL:!aNULL:!ADH:!eNULL:!LOW:!EXP:RC4+RSA:+HIGH:+MEDIUM
+ SSLProxyCipherSuite HIGH:MEDIUM:!MD5:!RC4
+ 
+ #  By the end of 2016, only TLSv1.2 ciphers should remain in use.
+@@ -83,7 +85,9 @@
+ #   Disable SSLv2 and SSLv3 by default (cf. RFC 7525 3.1.1).  TLSv1 (1.0)
+ #   should be disabled as quickly as practical.  By the end of 2016, only
+ #   the TLSv1.2 protocol or later should remain in use.
+-SSLProtocol all -SSLv2 -SSLv3
++# Limit SSL protocols.
++#SSLProtocol all -SSLv2 -SSLv3
++SSLProtocol -ALL +SSLv3 +TLSv1.2 +TLSv1
```

Comment:
SSLv3 should go at the end as that would override the TLSv1.2 option. As I understand SSLv3 and TLSv1 are basically the same thing.